scripts/start_fabric.sh

```diff
@@ -14,8 +14,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FABRIC_VERSION=1.4.4
+CHAIN_CODE_VERSION=1.0
```

Comment:
Please add a comment why we are using Fabric 1.4.4 version in Avalon as default instead of 2.0